Changes

```diff
@@ -2,6 +2,9 @@ Revision history for Audio::Scan
 
 Note: Bug numbers refer to bugs at http://bugs.slimdevices.com
 
+1.07	unreleased
```

Comment:
The version isn't changed in the corresponding `Scan.pm`. Shouldn't this be part of this PR?

Reply:
Done